### ISSUE TITLE
Whitespace around punctuation should use dotted space

### ISSUE THREAD

**Issue by dwaynebailey** [translate/pootle#5303]
This is fine tuning of the new editor, not critical but nice to have.

If you have `word[sp][punctuation]` then it is hard to see a space.  Currently we highlight trailing space:
![screen shot 2016-09-30 at 09 05 54](https://cloud.githubusercontent.com/assets/647438/18984955/6d79b2be-86ed-11e6-8a40-51b45cef028a.png)

But if we add a trailing punctuation as you would in French or other languages influenced by French you lose that cue.  As you can see here with `[sp]?` used in French for questions and `[sp]:` used in French for form labels. While mostly those would/should be `NBSP` there is the possibility of space here and in that case they are hard to see.
![screen shot 2016-09-30 at 09 06 07](https://cloud.githubusercontent.com/assets/647438/18984953/6d773750-86ed-11e6-930c-320c05e85905.png)

![screen shot 2016-09-30 at 09 06 21](https://cloud.githubusercontent.com/assets/647438/18984954/6d77dfd4-86ed-11e6-8fc8-21ed2166b7cf.png)

**Comment by iafan** [comment]
My thoughts on this: I don't think we (Evernote) will be adding this language-specific logic to the editor for the time being. We will better do this using some real-time checks that we plan to have in the future, and highlight differently, along with the explanation.

However, if you want to experiment with this, it should be as easy as adding a list of symbols inside the square bracket [into this line](https://github.com/translate/pootle/blob/a9c6450080d1a0c02e1e7cde5bdf6896f95368a4/pootle/static/js/editor/utils/font.js#L203), something like:

```JavaScript
newValue = newValue.replace(/ [\t\u00A0\.,:\?!]/g, surroundingSpaceReplacer);
```

**Comment by dwaynebailey** [comment]
Just ensuring the original post is clear, this is NOT language specific logic, its simply a place where stray whitespace can hide or be hidden.  The French example is simply a valid use case that exhibits the issue of whitespace around punctuation, it really isn't about the language.

**Comment by dwaynebailey** [comment]
Fixed in #5319
